features/src/main/feature/camel-features.xml

```diff
@@ -1357,6 +1357,20 @@
         <bundle>mvn:org.apache.camel.karaf/camel-attachments/${project.version}</bundle>
         <bundle>mvn:org.apache.camel.karaf/camel-mail/${project.version}</bundle>
     </feature>
+    <feature name='camel-mail-microsoft-oauth' version='${project.version}' start-level='50'>
+        <feature version='${camel.osgi.version.range}'>camel-mail</feature>
+        <feature version='[2.16,2.17)'>jackson</feature>
+        <bundle dependency='true'>wrap:mvn:com.microsoft.azure/msal4j/${msal4j-version}$overwrite=merge&amp;Import-Package=com.sun.net.httpserver;resolution:=optional,*</bundle>
```

Comment:
wrapped this bundle as it requires `com.sun.net.httpserver` which is a part of the jdk